src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Resources;
```

Comment:
@irrogozh yes you have the right references, but you need to create a ResourceManagment Client using the new nuget and you are still using the old client and recorded your test using the old client.